docs/readme/README.fr.md

```
@@ -0,0 +1,293 @@
+<img src="../../assets/rename-banner.svg" alt="Annonce : AltSendme devient DashBeam. La même application, plus facile à trouver, à retenir et à prononcer." width="1200" />
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)